doc/sphinx-guides/source/admin/dataverses-datasets.rst

```diff
@@ -0,0 +1,25 @@
+Managing Datasets and Dataverses
+================================
+
+.. contents:: |toctitle|
+	:local:
+
+Dataverses
+----------
+
+Move a Dataverse
+^^^^^^^^^^^^^^^^
+
+Moves a dataverse whose id is passed to a new dataverse whose id is passed. The dataverse alias also may be used instead of the id. If the moved dataverse has a guestbook, template, metadata block, or featured dataverse that is not compatible with the destination dataverse, you will be informed and given the option to force the move and remove the association. Only accessible to superusers. ::
+
+    POST http://$SERVER/api/dataverses/$id/move/$id?key=$apiKey
```

Comment:
Can you please replace this with a full `curl` command using the syntax is uses such as `-X POST`?